storage.rules

```diff
@@ -0,0 +1,7 @@
+service firebase.storage {
```

Comment:
do we need this?

Reply:
We currently do not need it. It does however let us configure it easily if we ever do need it in the future plus keeps the configuration in the repo so it cannot be changed without us knowing it.